crates/cxx-qt-lib/src/core/qstring.rs

```diff
@@ -114,6 +118,15 @@ impl fmt::Debug for QString {
     }
 }
 
+impl std::ops::Add for QString {
+    type Output = Self;
+    fn add(self, other: Self) -> Self {
+        let mut res = ffi::qstring_init_from_qstring(&self);
+        ffi::qstring_append(&mut res, &other);
```

Comment:
note that #416 adds the append method to QString, so i need to remember to update this part in that branch if that lands after...

Also note this was done without an FFI step via the following if you wanted to copy
```rust
/// Appends the string str onto the end of this string.
fn append<'a>(self: &'a mut QString, str: &QString) -> &'a mut QString;
````

Reply:
We can leave this one to #416 and not this merge :-)